jopa.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html>
+    <head>Hello world!</head>
```

Comment:
added line

Reply:
you have to add me to reviewers in this PR
![image](https://user-images.githubusercontent.com/9653011/79863958-4ebb4f00-83e1-11ea-8498-6b1265655627.png)

Reply:
ok, next.
you have to add "Hello, World" print not in HTML but in golang
you can check example [here](https://play.golang.org/p/NeviD0awXjt)
try it by Run.